méli-mélo/th-zev/zev.html

```diff
@@ -0,0 +1,180 @@
+---
+title: Zero Emission Vehicles (ZEV) Theme
+dateModified: 2023-04-20
+description: ZEV CSS styles
+lang: en
+css:
+- style.css
+layout: no-container
+---
+<div class="container">
+    <h1>{{ page.title }}</h1>
+    {% assign metadata = site.pages | where: "output", "false" | where: "componentName", "th-zev" | first %}
+    <p>Sponsor: {{ metadata.sponsor }}</p>
+    <p>Accessibility statement: {{ metadata.a11yStatement }}</p>
+    <p>Progressive enhancement note:</p>
+    <ul>
+      {% for peNote in metadata.peNote %}
+      <li>{{ peNote }}</li>
+      {% endfor %}
+    </ul>
+    <h2>On this page</h2>
+    <ul>
+      <li><a href="#a1">Coloured Backgrounds</a></li>
+      <li><a href="#a2">Example - Themed Wells</a></li>
+      <li><a href="#a3">Example - Themed Buttons</a></li>
+      <li><a href="#a4">Example - Themed Panels</a></li>
+    </ul>
+
+    <h2 id="a1">Coloured Backgrounds</h2>
+    <p>Used to theme an element with the green or purple background and text colors</p>
+    <div class="panel panel-default">
+      <div class="panel-body">
+        <h3 class="mrgn-tp-0">Appearance</h3>
+        <div class="row wb-eqht-grd mrgn-tp-md">
+          <div class="col-md-4">
+            <section class="well bg-zev-green well-sm">
+              <h4 class="h2 mrgn-tp-0">ZEV Green</h4>
+              <p>This section is styled with <code>.bg-zev-green</code> and <code>.well-sm</code>.</p>
+              <p><a class="well-link" href="#">Link example. Note the smaller padding on this well.</a></p>
```

Comment:
There is no "hover" effect (little color difference) like how other link behave. 

Note: I need to confirm if that is required but I suggest to add some.